frontend/src/pages/projects/screens/projects/ProjectView.stories.tsx

```diff
@@ -36,7 +39,9 @@ export default {
 
 const Template: ComponentStory<typeof ProjectView> = (args) => (
   <div data-testid="story-loaded">
-    <ProjectView {...args} />
+    <MemoryRouter>
+      <ProjectView {...args} />
+    </MemoryRouter>
```

Comment:
> i removed it because you cant have a MemoryRouter under a Router. And i needed a memory router defined story level because it takes a specific route structure. I will rewrite it so its more global